main.py

```diff
@@ -23,7 +23,6 @@ def get(self):
 		template_vals = {
 			'page': 'home'
 		}
-		
```

Comment:
Why?

Reply:
Incomplete revert, haven't cleaned it up yet.